core/templates/pages/profile-page/profile-page.mainpage.html

```diff
@@ -292,7 +292,7 @@
   <body>
     <oppia-root>
       <div ng-controller="Base">
-        <base-content>
+        <oppia-oppia-base-content>
```

Comment:
Typo? I'm assuming this & the closing tag should just be ``oppia-base-content``. Just to check: did you manually test all affected pages in a local dev server instance to make sure that everything is appearing correct after this change?

Reply:
Sorry, I just checked 2 pages, for others I planned to rely on e2e tests being passed.
But now, I will manually test all pages